crates/ln-dlc-node/src/tests/mod.rs

```diff
@@ -207,6 +238,7 @@ async fn fund_and_mine(address: bitcoin::Address, amount: bitcoin::Amount) {
 }
 
 /// Instructs `nigiri-chopsticks` to mine a block.
+#[allow(dead_code)]
```

Comment:
:wrench: We should delete this since we don't use `chopsticks` anymore.

Reply:
We still do. I created a follow up ticket for removing chopsticks. In the meantime I figured it would be best to keep that function.